src/components/elements/TabAddPublication/TabAddPublication.tsx

```diff
@@ -9,36 +9,40 @@ import Pagination from '../Pagination/Pagination';
 import Filter from '../Filter/Filter';
 import { YearPicker } from 'react-dropdown-date';
 
+
 interface FuncProps {
     onReject(id: number): void,
     onUndo(id: Number): void,
-    reciterData:any,
+    reciterData: any,
+    // updatePublicationAssertion: (reciterArticle: any, userAssertion: string, prevUserAssertion: string) => void
 }
 
 const TabAddPublication: FunctionComponent<FuncProps> = (props) => {
 
     const useStateWithCallback = (initialState: any) => {
         const [state, setState] = useState(initialState)
         const callbackRef = useRef(() => undefined)
```

Comment:
using Refs is not recommended in react, what are we trying to store: is it something that can be achieved by hooks only?

Reply:
Just removed the comments.  Ref is not my code change. Hence, did not touch that.

Reply:
I rebuilt most of the components since the old code was not written following the best practices and refactoring would have taken me a long time, thus if we are adding a component on the page that we are working on and it will be used in the new version of the app, we need to make sure that everything that goes in there makes sense.